pom.xml

```diff
@@ -197,6 +197,28 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+           <groupId>org.scalastyle</groupId>
+           <artifactId>scalastyle-maven-plugin</artifactId>
+           <version>1.0.0</version>
+           <configuration>
+             <verbose>false</verbose>
+             <failOnViolation>true</failOnViolation>
+             <includeTestSourceDirectory>true</includeTestSourceDirectory>
+             <failOnWarning>false</failOnWarning>
+             <sourceDirectory>${project.basedir}/src/main/scala</sourceDirectory>
+             <testSourceDirectory>${project.basedir}/src/test/scala</testSourceDirectory>
+             <configLocation>style/scalastyle-config.xml</configLocation>
+             <outputEncoding>UTF-8</outputEncoding>
+           </configuration>
+           <executions>
+             <execution>
+               <goals>
```

Comment:
Can you also add <phase>compile</phase> here so that scalastyle:check is run during mvn compile ? We do this for java checkstyle

Reply:
Done